src/runner/src/bin/main.rs

```diff
@@ -1,10 +1,47 @@
 use structopt::StructOpt;
-use fluvio_future::task::run_block_on;
 use fluvio_run::RunCmd;
 
+use tracing::Level;
+use tracing_subscriber::{Registry, prelude::*};
+use tracing_subscriber::filter::Directive;
+
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    fluvio_future::subscriber::init_tracer(None);
-    let cmd = RunCmd::from_args();
-    run_block_on(cmd.process())?;
+    let cmd: RunCmd = RunCmd::from_args();
+
+    #[cfg(feature = "telemetry")]
```

Comment:
use if_config to simplify

Reply:
I think if_cfg would actually be about the same as this setup, and it adds an external dependency. I'd like to leave it unless you have an overwhelmingly strong opinion about it